src/app/components/core/south/south-service-modal/south-service-modal.component.ts

```diff
@@ -251,30 +251,9 @@ export class SouthServiceModalComponent implements OnInit, OnChanges {
     this.child.toggleModal(true);
   }
 
-  shutdownService(svcInfo) {
-    if (this.isEnabled === false) {
-      this.deleteService(svcInfo.name);
-    } else {
-      this.servicesHealthService.shutDownService(svcInfo)
```

Comment:
`shutDownService` method should be removed from `servicesHealthService` as it is not required now.